var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -130,6 +130,9 @@ class Boost(Package):
     # Patch fix from https://svn.boost.org/trac/boost/ticket/11856
     patch('boost_11856.patch', when='@1.60.0%gcc@4.4.7')
 
+    # Patch fix from https://svn.boost.org/trac/boost/ticket/11120
+    patch('python_jam.patch')
```

Comment:
Ideally I would have specified an exact version range for which this patch works. However, either the Boost tarball is very large or my internet connection is slow today, so it took too long to download every version.